en-eng/Zigate-Plugin-FAQ.md

```diff
@@ -11,4 +11,12 @@ This is more an OS issue, but here are few hints
 1. Start the Plugin
 1. Trigger the pairing process from the device
 
+## I don't get the device updated in Domoticz when interacting via a remote control or manualy
+
+If I power Off an Ikea Bulb (with a physical switch), when switching it on I don't have a status updated in Domoticz. I have the same issue if using a remote controller .
+
+Most likely the issue is that the automatic reporting is not correctly configured at the device end. 
+
+1. Make sure that you have Allow reBinding Cluster in the PluginConf.txt ( that is the default ). check the parameters allowReBindingClusters it must be set to 1.
```

Comment:
1. Open the PluginConf.txt file (more information about this here  .....)
2. Please check if the parameter "AllowRebindingClusters" is set to **1**.
_If it is not, set it to "1_"